patches/minecraft/net/minecraft/client/gui/advancements/GuiAdvancementTab.java.patch

```diff
@@ -0,0 +1,47 @@
+--- ../src-base/minecraft/net/minecraft/client/gui/advancements/GuiAdvancementTab.java
++++ ../src-work/minecraft/net/minecraft/client/gui/advancements/GuiAdvancementTab.java
+@@ -37,7 +37,14 @@
+     private int field_191814_q = Integer.MIN_VALUE;
+     private float field_191815_r;
+     private boolean field_192992_s;
++    private int page;
+ 
++    public GuiAdvancementTab(Minecraft p_i47589_1_, GuiScreenAdvancements p_i47589_2_, AdvancementTabType p_i47589_3_, int p_i47589_4_, int page, Advancement p_i47589_5_, DisplayInfo p_i47589_6_)
++    {
++        this(p_i47589_1_, p_i47589_2_, p_i47589_3_, p_i47589_4_, p_i47589_5_, p_i47589_6_);
++        this.page = page;
++    }
++
+     public GuiAdvancementTab(Minecraft p_i47589_1_, GuiScreenAdvancements p_i47589_2_, AdvancementTabType p_i47589_3_, int p_i47589_4_, Advancement p_i47589_5_, DisplayInfo p_i47589_6_)
+     {
+         this.field_191802_a = p_i47589_1_;
+@@ -158,14 +165,16 @@
+         }
+         else
+         {
++            int page = p_193936_2_ / AdvancementTabType.field_192659_e;
++            int indexOnPage = p_193936_2_ % AdvancementTabType.field_192659_e;
```

Comment:
I think you could reuse the `p_193936_2_` variable here instead of introducing a new one (`indexOnPage`), doing that will eliminate a couple patch lines below.